packages/api/src/clickhouse/index.ts

```diff
@@ -1130,8 +1138,35 @@ const buildEventSeriesQuery = async ({
   const label = SqlString.escape(`${aggFn}(${field})`);
 
   const selectClause = [
-    isCountFn
+    aggFn === AggFn.Count
       ? 'toFloat64(count()) as data'
+      : aggFn === AggFn.CountPerSec
+      ? granularity
+        ? SqlString.format('divide(count(), ?) as data', [
```

Comment:
Fine for now, but I tend to think these kind of duplicates are safer as `CountPer('second')` or equivalent. Maybe a TODO here

Reply:
yeah, this select clause would probably need some refactoring later